Add Private Registries instructions to K8s SSI (#30453)
DataDog/documentation: content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm/kubernetes.md

## Advanced options

Use the following advanced options to customize how Single Step Instrumentation behaves in your environment. These settings are optional and typically only needed in specialized setups.

### Target specific workloads

> By default, SSI instruments all services in all namespaces in your cluster. To refine which services are instrumented and how, use one of the following configuration methods, depending on your Agent version. 

Reviewer: ```suggestion
By default, SSI instruments all services in all namespaces in your cluster. Depending on your Agent version, use one of the following configuration methods to refine which services are instrumented and how.
```

For instructions on changing your container registry, see [Changing Your Container Registry][33].

### Use a private container registry

If your organization does not allow direct pulls from public registries (such as `gcr.io`, `docker.io`, or `public.ecr.aws`), you can host the required Datadog images internally and configure the Admission Controller to use them.

To use SSI with a private container registry:

1. Follow [these instructions][34] to mirror Datadog's container images to your private registry. 

>    You need only the images for the languages you plan to instrument. If you're not sure which ones you'll need, here's a baseline that covers most use cases:

Reviewer: ```suggestion
   You only need the images for the languages you are instrumenting. If you're not sure which ones you need, here's a baseline that covers most use cases:
```
